blocks/carousel/carousel.css

```diff
@@ -0,0 +1,20 @@
+.carousel{
+  width: 100%;
+  margin-bottom: 2rem;
+}
+
+.carousel>ul{
+  display: flex;
+  flex-wrap: nowrap;
+  overflow-x: scroll;
+  gap: 1rem;
+}
+
+.carousel>ul>li{
+  min-width: 14rem;
+  background: #fff;
+  border-radius: 8px;
+  box-shadow: 0 1px 6px 1px rgba(64 64 64 / 15%);
```

Comment:
I see some opportunities to reuse the card CSS as well here.
Since the carousel cards have the very similar look and feel, you could simply load the CSS file from `blocks/cards/cards.css` instead of duplicating it. 

Then for the differences, you can write more specific CSS namespaced with `.carousel` in front, e.g. 
```css
.carousel .card {
 margin-right: 10px;
}
```